### PR TITLE
initial push

### PR DESCRIPTION
**Pull request by Priyatharshini-J**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+#Workflow will be started only if a tag is created with name starts with 'v'
+on:
+  push:
+    tags:
+      - "v*" # P
+jobs:
+  build:
+# Environment to load the secrets
+    environment: analyticsSync
```
Comment (akilavan-jayakumar) [inline .github/workflows/main.yml]: Change the environment name to **Production** and remove the comments

File: project-template-1.0.0.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    }, {
+      "type" : "function",
+      "name" : "zoho_analytics_datastore_sync_routes_handler",
+      "properties" : {
+        "stack" : "node16",
+        "code" : {
+          "path" : "functions/zoho_analytics_datastore_sync_routes_handler.zip"
+        },
+        "configuration" : {
+          "environment" : { },
```
Comment (akilavan-jayakumar) [inline project-template-1.0.0.json]: Environment variables needs to be updated in all the functions.

**Review by akilavan-jayakumar — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (Priyatharshini-J) [reply .github/workflows/main.yml]: Done

File: project-template-1.0.0.json (hunk 2 above)
Comment (Priyatharshini-J) [reply project-template-1.0.0.json]: Updated env var

**Pull request merged**